main.go

```diff
@@ -74,8 +75,8 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+	port := beego.AppConfig.DefaultInt("httpport", 80)
```

Comment:
We use the default port 8000. Can you avoid changing it?

Reply:
> We use the default port 8000. Can you avoid changing it?

@hsluoyz ok, I will change it back. And all the configuration will parsed from app.conf.

Reply:
@hsluoyz done